backend/src/main.py

```
@@ -1,32 +1,702 @@
+@app.errorhandler(500)
+def internal_error(error):
+    return jsonify({
+        'error': 'Internal server error',
+        'message': 'Something went wrong on the server'
+    }), 500
+
+# ============================================================================
+# RUN THE APP
+# ============================================================================
+
 if __name__ == '__main__':
```

Flake8 (7.2.0)

[error] expected 2 blank lines after class or function definition, found 1

(E305)